github-dark.css

```diff
@@ -1989,7 +1989,7 @@
   }
   .text-pending, a.text-pending, .compare-pr-placeholder p,
   .compare-pr-placeholder .help-link, .text-pending strong {
-    color: #cc5 !important;
+    color: #ddd !important;
   }
```

Comment:
Actually this particular definition doesn't touch the travis color (around line 1979).

https://github.com/jquery/globalize/pull/508#partial-pull-merging